plugins/mcreator-localization/help/default/entity/condition_is_body_visible.md

```diff
@@ -1,2 +1,2 @@
-If the returned value is true, the entity will be visible.
-When a player is in spectator mode, this value is false.
\ No newline at end of file
+While the returned value is true, the entity will be visible.
+While the player is in spectator mode, he does not meet this condition.
```

Comment:
We're talking about an abstract player, so this would look better:
```suggestion
While a player is in spectator mode, they do not meet this condition.
```

The same about `fr_FR` if supported.